tests/events/src/lib.rs

```diff
@@ -62,28 +63,14 @@ mod test {
         client.transfer(&from, &to, &amount);
 
         assert_eq!(
-            env.events().all(),
-            vec![
-                &env,
-                (
-                    contract_id.clone(),
-                    // Expect these event topics.
-                    (Symbol::new(&env, "transfer"), &from, to.address()).into_val(&env),
-                    // Expect this event body.
-                    map![
-                        &env,
-                        (
-                            symbol_short!("amount"),
-                            <_ as IntoVal<Env, Val>>::into_val(&1i128, &env)
-                        ),
-                        (
-                            Symbol::new(&env, "to_muxed_id"),
-                            <_ as IntoVal<Env, Val>>::into_val(&to.id().unwrap(), &env)
-                        ),
-                    ]
-                    .to_val()
-                ),
-            ],
+            env.events().contract_events(),
+            std::vec![Transfer {
```

Comment:
You shouldn't need a std::vec here, you should be able to use an array. It should mean you don't need the extern crate std either.
```suggestion
            [Transfer {
```
If that doesn't work, you should be able to use a slice:
```suggestion
            &[Transfer {
```